src/c/realize_c_type.c

```diff
@@ -384,6 +384,8 @@ _realize_c_struct_or_union(builder_c_t *builder, int sindex)
 
         if (!(s->flags & _CFFI_F_EXTERNAL)) {
             int flags = (s->flags & _CFFI_F_UNION) ? CT_UNION : CT_STRUCT;
+            int is_opaque = (s->flags & _CFFI_F_OPAQUE);
+            flags |= is_opaque ? CT_IS_OPAQUE : 0;
```

Comment:
Where is this from? It doesn't look to me like it has any effect as `new_struct_or_union_type` below always sets `CT_IS_OPAQUE`:

https://github.com/Quansight-Labs/cffi/blob/1101273ac870779440793042c98511c9c5ff4b5a/src/c/_cffi_backend.c#L5134

Reply:
I changed the line you linked to in this PR to unconditionally set `CT_UNDER_CONSTRUCTION`. Let me see if I make the change you suggested in your other comment if I can get rid of this.

Reply:
This is done now.